### PR TITLE
Make empty -doxycopy target warning visible when run.

### PR DESCRIPTION
**Pull request by hernando**

### CODE REVIEW

**Comment by bilgili** [comment]
+1

File: DoxygenRule.cmake [hunk 1]
```diff
@@ -186,11 +186,17 @@ if(GIT_DOCUMENTATION_REPO)
       COMMENT "Copying ${PROJECT_NAME} API documentation to ${GIT_DOCUMENTATION_DIR}"
       DEPENDS ${PROJECT_NAME}-doxygen VERBATIM)
   else()
+    # Having a command while force this target to run always when invoked. That
```
Comment (rdumusc) [inline DoxygenRule.cmake]: while -> will

File: DoxygenRule.cmake [hunk 2]
```diff
@@ -186,11 +186,17 @@ if(GIT_DOCUMENTATION_REPO)
...
+    # makes the Makefile to print the comment. The text will be also visible
+    # in ninja.
     add_custom_target(${PROJECT_NAME}-doxycopy
+      COMMAND ${CMAKE_COMMAND} -E echo ""
       COMMENT "doxycopy target not available, missing ${_GIT_DOC_SRC_DIR}")
```
Comment (rdumusc) [inline DoxygenRule.cmake]: What about merging COMMAND -E echo and COMMENT? Would it do the same..?

File: DoxygenRule.cmake (hunk 1 above)
Comment (hernando) [inline DoxygenRule.cmake]: I'm becoming dyslexic.. the more I write in English the more I made this word substitution mistakes.

**Comment by rdumusc** [comment]
I'm not against this change, but I still wonder under which conditions this message is not printed. As far as I can remember this has always worked for me:
make DisplayCluster-doxycopy
Scanning dependencies of target DisplayCluster-doxycopy
doxycopy target not available, missing /home/dumusc/src/stable/DisplayCluster/../BlueBrain
Built target DisplayCluster-doxycopy

File: DoxygenRule.cmake (hunk 2 above)
Comment (hernando) [inline DoxygenRule.cmake]: Not sure. It looks nicer like this with the Makefile generator because it stands out with blue ink.

**Comment by hernando** [comment]
It's a CMake 2 vs 3 version. With CMake 2 it gets printed, not with 3. Probably CMake 3 detects the empty target and generates less code.

**Comment by rdumusc** [comment]
hah! OK, +1 for me

**Pull request merged**